HotelBooking.API/Controllers/HotelController.cs

```diff
@@ -68,6 +68,20 @@ public async Task<IActionResult> FilterHotel(FilterHotelRequest filter)
             });
         }
 
+        [HttpGet("get-hotel-by-name")]
+        public async Task<IActionResult> GetHotelByNameAsync(string name)
+        {
+            if (string.IsNullOrEmpty(name))
```

Comment:
we should getAll if name is empty

Reply:
I will fix it when I add GetAllHotel func. Thanks